xs/src/libslic3r/ConfigBase.hpp

```diff
@@ -317,9 +351,19 @@ class ConfigOptionFloatOrPercent : public ConfigOptionPercent
     ConfigOptionFloatOrPercent() : ConfigOptionPercent(0), percent(false) {};
```

Comment:
Since `ConfigOptionPercent` inherits off of `ConfigOptionFloat`, was it intended to have `getFloat()` return a bad option type or use the default behavior (return the floating point value)?
Between leaving it from `ConfigOptionFloat`'s behavior and then overriding it in `ConfigOptionFloatOrPercent`, I am unsure about the intended behavior here.

Reply:
It's wanted, as `getFloat()` cannot return the `percent` flag, so the semantics of the returned float are totally ambiguous...

Reply:
So ConfigOptionPercent supports getFloat, then?